application/models/Eqslmethods_model.php

```diff
@@ -41,15 +41,15 @@ function downloadUser($userid, $username, $password) {
 
 	function uploadUser($userid, $username, $password) {
 		$data['user_eqsl_name'] = $this->security->xss_clean($username);
-		$data['user_eqsl_password'] = html_entity_decode($this->security->xss_clean($password));
```

Comment:
this was added by me 2 months ago in commit 7c82d51916ae0f20d6d65990cd608b4b704c823c

reason was that special chars in the password caused the upload to break. See PR https://github.com/wavelog/wavelog/pull/696

this may solves the UTF8 issue but brings back the password encoding issue

Reply:
corresponding issue https://github.com/wavelog/wavelog/issues/695

Reply:
Tested this, too.
Characters like `<` or `>` are converted with the new `rawurlencode` at ADIF-Generation.